src/__tests__/Box.spec.tsx

```diff
@@ -13,49 +13,4 @@ describe('Box component', () => {
     expect(ref).toHaveBeenLastCalledWith(expect.any(Object));
     wrapper.unmount();
   });
-
-  it('passes all props but hidden and responsive', () => {
```

Comment:
It must have been erroneously committed one day. Equal tests are present in Image.spec.tsx